massa-signature/src/error.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use displaydoc::Display;
+use thiserror::Error;
+
+#[non_exhaustive]
+#[derive(Display, Error, Debug)]
+pub enum MassaSignatureError {
+    /// parsing error : {0}
+    ParsingError(String),
+
+    /// error forwarded by engine: {0}
```

Comment:
```suggestion
    /// secp256k1 engine error: {0}
```

Reply:
It's schnorr engine no ?

Reply:
good question I took the crate name

Reply:
saying secp256k1 engine error is the right thing to do, schnorr is just a part of what is handled by the crate engine